app/src/main/res/layout/activity_login.xml

```diff
@@ -36,7 +35,9 @@
         app:layout_constraintBottom_toTopOf="@+id/log_button_sub"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textInfo2" />
+        app:layout_constraintTop_toBottomOf="@+id/textInfo2"
+        android:autofillHints="Password"
+        tools:ignore="LabelFor" />
```

Comment:
I don't think we want to ignore LabelFor on basic interface elements. They're necessary for accessibility. ignore="LabelFor" is here for stuff like logo or design elements.